doc/classes/Array.xml

```diff
@@ -297,88 +303,83 @@
 			<param index="0" name="what" type="Variant" />
 			<param index="1" name="from" type="int" default="0" />
 			<description>
-				Searches the array for a value and returns its index or [code]-1[/code] if not found. Optionally, the initial search index can be passed.
+				Returns the index of the [b]first[/b] occurrence of [param what] in this array, or [code]-1[/code] if there are none. The search's start can be specified with [param from], continuing to the end of the array.
+				[b]Note:[/b] If you just want to know whether the array contains [param what], use [method has] ([code]Contains[/code] in C#). In GDScript, you may also use the [code]in[/code] operator.
 			</description>
 		</method>
 		<method name="front" qualifiers="const">
 			<return type="Variant" />
 			<description>
-				Returns the first element of the array. Prints an error and returns [code]null[/code] if the array is empty.
-				[b]Note:[/b] Calling this function is not the same as writing [code]array[0][/code]. If the array is empty, accessing by index will pause project execution when running from the editor.
+				?? Returns the first element of the array. If the array is empty, generates an error and returns [code]null[/code], similarly to [], without stopping the project execution.
+				?? Returns the first element of the array. If the array is empty, generates an error and returns [code]null[/code] without stopping the project execution (unlike with the [code][][/code] operator).
```

Comment:
Same as above.
Unsure of how to say this, here. I do not want to lose "Calling this function is not the same as writing [code]array[0][/code]", but it also feels like repetitive information at the same time.

Reply:
If you had to pick from these options, the second line is a lot more clear at demonstrating the point. 
The use of `unlike` clarifies how the function will do something different than regularly accessing the array.
The first line uses `similarly`, which implies neither stop project execution, which gives the wrong impression.